### DISCUSSION TITLE
请求帮助

### DISCUSSION THREAD

**Discussion by bush9909** [community/community#57275] [Programming Help]
### Select Topic Area

Question

### Body

Microsoft Windows [版本 10.0.19043.2364]
(c) Microsoft Corporation。保留所有权利。

C:\Users\Administrator>pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
Looking in indexes: https://download.pytorch.org/whl/cu118
Requirement already satisfied: torch in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (2.0.1+cu118)
Requirement already satisfied: torchvision in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (0.15.2+cu118)
Requirement already satisfied: torchaudio in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (2.0.2+cu118)
Requirement already satisfied: filelock in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torch) (3.9.0)
Requirement already satisfied: typing-extensions in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torch) (4.4.0)
Requirement already satisfied: sympy in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torch) (1.11.1)
Requirement already satisfied: networkx in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torch) (3.0)
Requirement already satisfied: jinja2 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torch) (3.1.2)
Requirement already satisfied: numpy in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torchvision) (1.24.1)
Requirement already satisfied: requests in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torchvision) (2.28.1)
Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from torchvision) (9.3.0)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from jinja2->torch) (2.1.2)
Requirement already satisfied: charset-normalizer<3,>=2 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from requests->torchvision) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from requests->torchvision) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from requests->torchvision) (1.26.13)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from requests->torchvision) (2022.12.7)
Requirement already satisfied: mpmath>=0.19 in c:\users\administrator\appdata\local\programs\python\python311\lib\site-packages (from sympy->torch) (1.2.1)

C:\Users\Administrator>
什么情况，这是安装完了，不能运行

**Comment by emilburganov** [comment]
https://stackoverflow.com/questions/45673949/error-importing-pytorch-python

**Reply by bush9909** [reply]
who can help me? Thanks!
